.github/workflows/test-clib.yaml

```diff
@@ -34,7 +34,8 @@ jobs:
         uses: goto-bus-stop/setup-zig@v2
         with:
           cache: false
-      # TODO: Broken so disabling — https://github.com/PRQL/prql/issues/4050
-      # - name: Run example minimal-zig
-      #   working-directory: prqlc/bindings/clib/examples/minimal-zig
-      #   run: make run test
+          # Pinned version — https://github.com/PRQL/prql/issues/4050
+          version: 0.11.0
```

Comment:
Just a question: When is this version supposed to be updated and by whom?
For example, the version of Elixir is specified in the same way, but it has not been updated in two years and is probably currently inappropriate.

https://github.com/PRQL/prql/blob/eaf7839e57c827d10c85b4498de9f906c55a213b/.github/workflows/test-elixir.yaml#L27-L28

In my opinion it would be useful to be able to specify the latest release here. (This is hacky but I think we can use a script to set the latest version here)

Reply:
Good question, I don't have any good answer.
I agree that it would be  useful to be able to specify the latest release but the `goto-bus-stop/setup-zig` GitHub Action does not have any support for that.